### ISSUE TITLE
Unused interpolated string (with nothing to interpolate) should produce warning similar to unused string literal

### ISSUE THREAD

**Issue by Youssef1313** [dotnet/roslyn#53226]
```csharp
var x = $"Some string";
```

**Actual Behavior**:
I think the previous code is transformed into a `String.Format` method call, even if there are no `{ }` between the double quotes.

**Expected Behavior**:
It's better if the `$` is just ignored and the code is treated as

```csharp
var x = "Some string";
```

====

**Update:**

As pointed by @huoyaoyuan , The compiler seem to really ignore the `$` when there is nothing to interpolate. but, the following two lines behave differently for warnings:

```csharp
void Main()
{
	var x1 = "abc"; // Warning: CS0219
	var x2 = $"abc"; // No-warnings.
}
```
